cmake_modules/vcpkg.cmake

```diff
@@ -19,8 +19,15 @@ elseif(APPLE)
     set(VCPKG_TARGET_TRIPLET "x64-osx" CACHE STRING "target triplet")
   endif()
 elseif(UNIX)
+  execute_process(COMMAND uname -m OUTPUT_VARIABLE ARCHITECTURE)
```

Comment:
Aside: If we ever want to do cross-compiling, this won't be correct.

Reply:
Agree - I had a mental note to see if there was a better way than this (like the OSX flag).